rubin_jupyter_utils/hub/scanrepo/parse_args.py

```diff
@@ -67,6 +67,12 @@ def parse_args(
             help="repository owner [{}]".format(lro),
             default=lro,
         )
+        parser.add_argument("--username",
+                            help="Docker username",
```

Comment:
Likewise, "Docker repo username"; otherwise it sounds a lot like `the name of the user running the Docker daemon` which isn't going to be helpful.